### ISSUE TITLE
`val_acc` which is not available

### ISSUE THREAD

**Issue by Akeepers** [finlay-liu/kaggle_public#5]
>  RuntimeWarning: Early stopping conditioned on metric `val_acc` which is not available. Available metrics are: val_loss,val_accuracy,val_acc_top2,loss,accuracy,acc_top2
  (self.monitor, ','.join(list(logs.keys()))), RuntimeWarning

出现这个错是因为什么啊？keras版本？需要将val_acc改成val_accuracy 或者是val_loss吗

**Comment by finlay-liu** [comment]
你吧出错信息谷歌搜下吧，我没遇到这个错误。
